src/cidr/ip_cidr.rs

```diff
@@ -17,6 +17,7 @@ pub enum IpCidr {
 
 impl IpCidr {
     #[allow(clippy::should_implement_trait)]
+    #[doc(hidden)] // TODO: remove and rely on std FromStr
```

Comment:
Is this function necessary to be hidden?

Reply:
We should encourage use of the FromStr trait. If a breaking change were acceptable, I'd say to make this fn private. Hiding it is an alternative that is backwards compatible.

Reply:
It is more convenient that a type has `from_str` itself. We don't need to add the `use std::str::FromStr` statement every time.

Reply:
Using `str::parse()` is the recommended method, not `FromStr`, which is why it's not in the prelude.

Reply:
removed this particular change from this PR